README.md

```diff
@@ -1,14 +1,37 @@
 # CAKE with Adaptive Bandwidth - "autorate"
 
-**CAKE-autorate** is a script that automatically adapts [CAKE Smart Queue Management (SQM)](https://www.bufferbloat.net/projects/codel/wiki/Cake/) bandwidth settings by measuring traffic load and RTT times. This is designed for variable bandwidth connections such as LTE, **and is not intended for use on connections that have a stable, fixed bandwidth**.
-
-CAKE is an algorithm that manages the buffering of data being sent/received by a device such as an [OpenWrt router](https://openwrt.org) so that no more data is queued than is necessary, minimizing the latency ("bufferbloat") and improving the responsiveness of a network.
+**CAKE-autorate** is a script that automatically adapts
```

Comment:
Why have you split up all the lines by adding newlines here? I believe GitHub adds the new lines itself as appropriate for the browser window size? Or am I mistaken?

Reply:
It's my personal peccadillo. But this practice is also enforced by [markdown-lint](https://github.com/markdownlint/markdownlint). I'll admit this technique screws up the git diff one time, but then it makes it better forever. 

That is, without these line breaks, it's really hard to see the difference in a paragraph of a dozen lines - the change is _way_ out on the right margin (or if the diff utility wraps the lines, it's still often hard to see the changes.)

If the lines are all ~80 characters or so, the changed lines pop out at you, whether a single word, or a couple sentences are dropped/added. And if you do remove an entire paragraph, at least all the content is visible in the diff so you can ensure you've still captured any important ideas.